public/chore.js

```diff
@@ -30,4 +30,11 @@ choreList.forEach(function (chore) {
 });
 
 const app = document.querySelector('#app');
-app.appendChild(list);
\ No newline at end of file
+app.appendChild(list);
+
+function addChore() {
+    newChore = document.getElementById('chore').value;
```

Comment:
![image](https://user-images.githubusercontent.com/3535669/81067566-77911900-8ea4-11ea-9e1e-3434bf180b7c.png)

Right now when I fill out the Chore and Assignee, only the chore is pushed to the list. You need to grab the chore and the assignee and map them to an object that matches the structure of the other chore objects in the choreList array.